sequencer/node/src/node.rs

```diff
@@ -143,12 +150,17 @@ impl Node {
 
                             let n = 10_usize;
                             let program = include_bytes!("../../cairo_programs/fib_contract.casm");
-                            let ret = run_cairo_1_entrypoint(
-                                program.as_slice(),
-                                0,
-                                &[0_usize.into(), 1_usize.into(), n.into()],
-                            );
-                            info!("Output: ret is {:?}", ret);
+                            let execution_engine = true;
```

Comment:
If possible it would be nice to have this be an argument for the binary. If not, an env var would work just fine as well.

Reply:
This will be added in a separated PR. It would be sent as a parameter when the node is spin up.